RNTableView/RNTableView.m

```diff
@@ -172,9 +172,10 @@ - (void)layoutSubviews {
 - (NSArray *)sectionIndexTitlesForTableView:(UITableView *)tableView {
     // create selected indexes
     NSMutableArray *keys = [NSMutableArray arrayWithCapacity:[_sections count]];
-
-    for (NSDictionary *section in _sections){
-        [keys addObject:section[@"label"]];
+    if (_sectionIndexTitlesEnabled) {
```

Comment:
Minor thing... since `sectionIndexTitlesEnabled` is a property, better off accessing it instead of the ivar directly.